src/components/Calculator.js

```diff
@@ -1,38 +1,53 @@
-          <button type="button" className="calc-btn">5</button>
-          <button type="button" className="calc-btn">6</button>
-          <button type="button" className="calc-btn calculation-btn">-</button>
-        </div>
-        <div>
-          <button type="button" className="calc-btn">1</button>
-          <button type="button" className="calc-btn">2</button>
-          <button type="button" className="calc-btn">3</button>
-          <button type="button" className="calc-btn calculation-btn">+</button>
+      <div className="calculator-container">
+        <div className="calculator-result">
+          {currentState.total}
+          {currentState.operation}
+          {currentState.next}
         </div>
-        <div>
-          <button type="button" className="calc-btn zero-btn">0</button>
-          <button type="button" className="calc-btn">.</button>
-          <button type="button" className="calc-btn calculation-btn">=</button>
+        <div className="calculator-keypad">
+          <div>
+            <button type="button" className="calc-btn" onClick={handleClick}>AC</button>
+            <button type="button" className="calc-btn" onClick={handleClick}>+/-</button>
+            <button type="button" className="calc-btn" onClick={handleClick}>%</button>
+            <button type="button" className="calc-btn calculation-btn" onClick={handleClick}>÷</button>
+          </div>
+          <div>
+            <button type="button" className="calc-btn" onClick={handleClick}>7</button>
+            <button type="button" className="calc-btn" onClick={handleClick}>8</button>
+            <button type="button" className="calc-btn" onClick={handleClick}>9</button>
+            <button type="button" className="calc-btn calculation-btn" onClick={handleClick}>x</button>
+          </div>
+          <div>
+            <button type="button" className="calc-btn" onClick={handleClick}>4</button>
+            <button type="button" className="calc-btn" onClick={handleClick}>5</button>
+            <button type="button" className="calc-btn" onClick={handleClick}>6</button>
+            <button type="button" className="calc-btn calculation-btn" onClick={handleClick}>-</button>
+          </div>
+          <div>
+            <button type="button" className="calc-btn" onClick={handleClick}>1</button>
+            <button type="button" className="calc-btn" onClick={handleClick}>2</button>
+            <button type="button" className="calc-btn" onClick={handleClick}>3</button>
+            <button type="button" className="calc-btn calculation-btn" onClick={handleClick}>+</button>
+          </div>
+          <div>
+            <button type="button" className="calc-btn zero-btn" onClick={handleClick}>0</button>
+            <button type="button" className="calc-btn" onClick={handleClick}>.</button>
+            <button type="button" className="calc-btn calculation-btn" onClick={handleClick}>=</button>
+          </div>
         </div>
       </div>
```

Comment:
- [ ] You were required to create a child component in a previous project. Now in this project, you are required to pass props to the child component. I cannot find any child components here. Kindly create a child component and use it as a child of this calculator component in any way you deem fit, but the state should remain in this Calculator component.

Reply:
@julie-ify Hello! I'll really appreciate it if you can clear some doubts about this project. 
I think I've built a child component already. This child component is created in another file, named CalculatorComponent.js
That's why you are not going to find the child in this Calculator.js, Can you check the CalculatorComponent.js, please? So you can tell me if this is okay

Reply:
@julie-ify Here are some screenshots of what I'm talking about
![event-react-1](https://user-images.githubusercontent.com/105683031/234654108-035209ce-4375-44f7-88db-99fba472ee3e.png)
![react-event-2](https://user-images.githubusercontent.com/105683031/234654127-ee78f6be-ca24-4c53-b516-6c46bf564043.png)

Reply:
@julie-ify Or I'm supposed to do it like this?
![react-event-3](https://user-images.githubusercontent.com/105683031/234655061-f439a907-eb1b-4e85-a255-680e18acc44c.png)
![react-event-4](https://user-images.githubusercontent.com/105683031/234655066-a4a90510-5aea-4e76-92e2-8537bbfee1a6.png)